### ISSUE TITLE
Chalice local only works for API and not for s3Event

### ISSUE THREAD

**Issue by Rubyj** [aws/chalice#1118]
@joguSD @dstufft @stealthycoin @jamesls @kyleknap Can we please have a form of chalice local that allows us to run a lambda function locally that isn't set to spin up a local server? I know #900 proposes having invoke for a local function. I would argue this is necessary behavior to accurate test an application before deploying.

I.E. I have a `on_s3_event` I would like to be able to test locally by feeding it a test event and running.

This may be a duplicate of #899

**Comment by joguSD** [comment]
Totally agree with you. We don't have a local implementation of the `invoke` command yet. This is a duplicate of #899. We should have a local version of invoke as it at least unblocks basic testing of both pure lambdas, and lambda attached to events (though it might be cumbersome). 

Closing as duplicate of #899.
